erpnext/accounts/report/item_wise_purchase_register/item_wise_purchase_register.py

```diff
@@ -87,7 +87,7 @@ def _execute(filters=None, additional_table_columns=None, additional_query_colum
 				"project": d.project,
 				"company": d.company,
 				"purchase_order": d.purchase_order,
-				"purchase_receipt": d.purchase_receipt,
+				"purchase_receipt": purchase_receipt,
```

Comment:
I'm not completely sure about this change.

@deepeshgarg007 could you please take a look?